moc_openshift.py

```diff
@@ -447,6 +447,45 @@ def update_rolebindings(self, project_name, role, rolebindings_json):
         self.logger.debug("payload -> 2: " + json.dumps(payload))
         return self.put_request(url, payload, True)
 
+    # member functions for quotas
+    #Check if Quota exists
+    def quota_exists(self, project_name, resource_name):
+            url = f"{self.get_url()}/api/v1/namespaces/{project_name}/resourcequotas/{resource_name}"
+            result = self.get_request(url, True)
+            if result.status_code == 200 or result.status_code == 201:
+                return True
+            return False
+
+    #create quotas, (object_def) contains the specification to be set for particular quota
```

Comment:
when you do want to add some description to a method and its arguments, you should use [docstrings](https://www.python.org/dev/peps/pep-0257/#id15) in python instead of a comment on top.

You can, of course, still add comments when needed.